RecoEgamma/ElectronIdentification/test/runtests.sh

```diff
@@ -1,14 +1,16 @@
 function die { echo $1: status $2 ;  exit $2; }
 
 ids_to_test=(
-"RecoEgamma.ElectronIdentification.Identification.cutBasedElectronID_PHYS14_PU20bx25_V2_cff"
-"RecoEgamma.ElectronIdentification.Identification.cutBasedElectronID_Spring15_25ns_V1_cff"
-"RecoEgamma.ElectronIdentification.Identification.cutBasedElectronID_Spring15_50ns_V2_cff"
-"RecoEgamma.ElectronIdentification.Identification.heepElectronID_HEEPV60_cff"
-"RecoEgamma.ElectronIdentification.Identification.heepElectronID_HEEPV70_cff"
```

Comment:
this is apparently still used in the nanoAOD.
Please restore or clarify why it can be removed.